templates/deployment-zabbix-web.yaml

```diff
@@ -51,7 +51,11 @@ spec:
           {{- toYaml .Values.zabbixweb.resources | nindent 10 }}
         securityContext:
           {{- toYaml .Values.securityContext | nindent 12 }}
+        {{- if .Values.zabbixweb_image.tag }}
```

Comment:
Hi @scoopex!

Can you change

{{- if .Values.zabbixweb_image.tag }}

by

{{- if .Values.zabbixweb.image.tag }}

?

Reply:
Sure, done :-)